lib/repository/patients.repository.js

```diff
@@ -278,6 +280,28 @@ const repository = {
       .promise()
       .then((result) => result.Item)
   },
+
+  async informFacilityDoctors(facilityName, patient) {
```

Comment:
Update your branch with `development` before pushing. This `informFacilityDoctors` function was already removed from this repository.

An easy way of updating your branches would be `git pull --rebase origin development`